extensions/wikia/FounderEmails/events/FounderEmailsViewsDigestEvent.class.php

```diff
@@ -40,14 +38,22 @@ public function process ( Array $events ) {
 		foreach ( $cityList as $cityID ) {
 			Wikia::initAsyncRequest( $cityID );
 			$userIds = $wikiService->getWikiAdminIds( $cityID );
+			$views = $founderEmailObj->getPageViews( $cityID );
+
+			// Don't bother sending this email if there are no page views for this period
+			if ( empty( $views ) ) {
+				continue;
+			}
+
 			$emailParams = [
-				'pageViews' => $founderEmailObj->getPageViews( $cityID )
+				'pageViews' => $views,
+				'wikiId' => $cityID,
```

Comment:
This is the real bug of this PR; ```wikiId``` is required field but it wasn't being passed in.